person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private_class_method def self.of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @age >= 18 && @parent_permission
+  end
```

Comment:
- [ ] Great job here 💯 However, according to the requirements `can_use_services?` should return true if person is of age **or** if they have permission from parents. (not and), so Kindly fix this small issue 👍

![image](https://user-images.githubusercontent.com/45231276/195280522-922259c9-9ba5-44c9-875f-23d94f647ec2.png)